kotlin-analysis-api/src/main/kotlin/com/google/devtools/ksp/impl/symbol/kotlin/KSFileImpl.kt

```diff
@@ -93,4 +93,15 @@ class KSFileImpl private constructor(private val ktFileSymbol: KtFileSymbol) : K
     override fun toString(): String {
         return "File: ${this.fileName}"
     }
+
+    override fun defer(): Restorable {
+        val psi = this.psi
+        return Restorable {
+            when (psi) {
+                is KtFile -> analyze { getCached(psi.getFileSymbol()) }
+                is PsiJavaFile -> null
```

Comment:
when will we hit this line now that we have separate KSFileJavaImpl for java files?